### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amitkumarj441**
**- What I did**
Linked Docker API link
**- How I did it**
Reference: Docker Remote API Guide
**- How to verify it**
Same as 2
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

Linked Docker API link

### CODE REVIEW

**Comment by amitkumarj441** [comment]
Thanks @GordonTheTurtle for letting me know about this info!

**Comment by amitkumarj441** [comment]
@GordonTheTurtle I just signed-off my commits, please do review this!

**Comment by cpuguy83** [comment]
@amitkumarj441 Need to do something like so:

```
$ git reset --hard HEAD~1
$ git commit -s --amend --no-edit
$ git push -f origin <branch>
```

This will remove the 2nd (empty) commit, sign the first commit, then force push to the branch.

**Comment by vieux** [comment]
ping @amitkumarj441

**Comment by amitkumarj441** [comment]
Hey @vieux ,

Apologize for the delay!
I got buried into Linux Conference and now I'm back to my work.
I'll be updating this PR very soon. Please do let me know if I can add anything more in this!

Cheers,
Amit Kumar Jaiswal

File: api/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This directory contains code pertaining to the Docker API:
+This directory contains code pertaining to the [Docker API](https://docker.github.io/engine/reference/api/docker_remote_api/):
```
Comment (amitkumarj441) [inline api/README.md]: @thaJeztah Alright! 
But we can update it here also as it's a part of Docker Docs!
Should I update it here itself or not?

**Comment by vdemeester** [comment]
ping @amitkumarj441 : the commit https://github.com/docker/docker/pull/28028/commits/da04587099d926d34ad0707a5039343b944b514c is not signed-off. Best would be to squash those two. Could you also take care of @thaJeztah's comment ?

**Comment by thaJeztah** [comment]
This URL is also going to change soon due to the switch to Swagger. Let me close this one for now until that's settled

**Pull request closed without merging**